### PR TITLE
noetic-devel

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
Is compatible with older distributions.

Merge together with: https://github.com/tue-robotics/hero_bringup/pull/80

Cleanup PR in #216

### CODE REVIEW

**Comment by ar13pit** [comment]
@MatthijsBurgh It will warrant way too much cleanup later on to remove older distros

File: catkin/install.yaml [hunk 1]
```diff
@@ -2,4 +2,9 @@
   name: ros-setup
 
 - type: target
-  name: python-catkin-tools
+  default:
+    name: python3-catkin-tools
```
Comment (Timple) [inline catkin/install.yaml:6]: I think this one still requires `python3-osrf-pycommon` no? The 'common' workaround in noetic.

Comment (MatthijsBurgh) [reply catkin/install.yaml:6]: Yep, this is a target dependency. In the target, it is included.

Comment (Timple) [reply catkin/install.yaml:6]: Ah, correct!

**Review by ar13pit — DISMISSED** [review]

**Review by ar13pit — APPROVED** [review]

**Pull request merged**